pegasus/sites.v3/hourofcode.com/public/index.haml

```diff
@@ -232,3 +245,31 @@ social:
       caption.fadeOut();
     });
   });
+
+  function startLATAMVideo()
+  {
+    if (videoStarted)
+      return;
+
+    $("<iframe allowfullscreen frameborder='0' src='https://www.youtube-nocookie.com/embed/OpgWbXthMlQ?iv_load_policy=3&amp;rel=0&amp;autohide=1&amp;showinfo=0&amp;autoplay=1&cc_load_policy=1' style='position:absolute; top: 0; left: 0; width: 100%; height: 100%; background-color:black;'></iframe>").appendTo("#videodiv");
+
+    $("#videodiv").fadeTo(1000, 1);
+    $("#leftbutton").fadeTo(500, 0).remove();
+
+    $("#fullwidth" + currentImage).fadeTo(1000, 0.3);
+
+    videoStarted = true;
+
+    clearInterval(timerId);
+  }
+  $(function() {
+    $('.highlight-item').mouseover(function() {
```

Comment:
Do we expect the caption to be available on touch devices in some way?

Reply:
How come this part has to be repeated even though it's exactly the same as the original video and it's outside of the startLATAMVideo() function? (I may just not understand Javascript)

```
$(function() {
    $('.highlight-item').mouseover(function() {
      var caption = $(this).find('.highlight-caption');
      caption.fadeIn();
    });

    $('.highlight-item').mouseleave(function() {
      var caption = $(this).find('.highlight-caption');
      caption.fadeOut();
    });
  });
```